src/core_modules/capture-core/components/FormFields/New/Fields/DateAndTimeFields/DateField/DateField.component.js

```diff
@@ -1,47 +1,81 @@
 // @flow
 import * as React from 'react';
 import { withStyles, withTheme } from '@material-ui/core/styles';
-import { DateField as UIDateField } from 'capture-ui';
+import { IconButton, DateField as UIDateField, orientations } from 'capture-ui';
+import { IconCross24 } from '@dhis2/ui';
+import classNames from 'classnames';
 import { systemSettingsStore } from '../../../../../../metaDataMemoryStores';
 
-const getStyles = (theme: Theme) => ({
-    innerInputError: {
-        color: theme.palette.error.main,
-        padding: theme.typography.pxToRem(3),
-        fontSize: theme.typography.pxToRem(12),
+const getStyles = () => ({
+    fieldsContainer: {
+        display: 'flex',
     },
-    innerInputWarning: {
-        color: theme.palette.warning.dark,
-        padding: theme.typography.pxToRem(3),
-        fontSize: theme.typography.pxToRem(12),
+    fieldsContainerVertical: {
+        flexDirection: 'column-reverse',
+        alignItems: 'center',
     },
-    innerInputInfo: {
-        color: 'green',
-        padding: theme.typography.pxToRem(3),
-        fontSize: theme.typography.pxToRem(12),
-    },
-    innerInputValidating: {
-        color: 'orange',
-        padding: theme.typography.pxToRem(3),
-        fontSize: theme.typography.pxToRem(12),
+    clearButton: {
+        height: '40px',
+        width: '40px',
+        borderRadius: 0,
     },
 });
 
 type Props = {
-}
+    value?: any,
+    disabled?: boolean,
+    onBlur?: (value: any) => void,
+    orientation: string,
+    classes: {
+        fieldsContainer: string,
+        fieldsContainerVertical: string,
+        clearButton: string,
+    },
+};
+
+const DateFieldPlain = (props: Props) => {
+    const {
+        value,
+        onBlur,
+        disabled,
+        orientation,
```

Comment:
Since it was previously part of passOnProps, could you please explicitly pass `disabled` to the `UIDateField` component? I'm not sure if `orientation` is also necessary down the line, but `disabled` definitely is (e.g., when we use a program rule to assign a value, we'll disable the assigned value field). Thank you!